manifests/kustomize/third-party/argo/installs/cluster/kustomization.yaml

```
@@ -1,12 +1,9 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-  # Note, we do not explicitly separate cluster-scoped resources for cluster installation,
-  # because people who deploy cluster-scoped resources should be the same as who deploys
-  # namespaced resources.
 resources:
-- https://github.com/argoproj/argo-workflows/manifests/base/crds?ref=v3.6.7
-- https://github.com/argoproj/argo-workflows/manifests/cluster-install/workflow-controller-rbac?ref=v3.6.7
+- https://github.com/argoproj/argo-workflows/manifests/base/crds?ref=v3.7.0
+- https://github.com/argoproj/argo-workflows/manifests/cluster-install/workflow-controller-rbac?ref=v3.7.0
```

looks like argo shuffled their manifests a bit in v3.7.0, which is why `kustomize build` fails to find these remote refs.  I think this should fix it:  

```suggestion
- https://github.com/argoproj/argo-workflows/manifests/base/crds/minimal?ref=v3.7.0
- https://github.com/argoproj/argo-workflows/manifests/cluster-install-no-crds/workflow-controller-rbac?ref=v3.7.0

```